components/ws-proxy/pkg/proxy/routes.go

```diff
@@ -434,6 +437,11 @@ func installWorkspacePortRoutes(r *mux.Router, config *RouteHandlerConfig, infoP
 				workspacePodPortResolver,
 				withHTTPErrorHandler(showPortNotFoundPage),
 				withXFrameOptionsFilter(),
+				func(h *proxyPassConfig) {
+					h.Transport = &http.Transport{
```

Comment:
@iQQBot is it possible to make it implicit? i.e. try to do http request first and then it fails with TLS then try https? cc @csweichel 

@loujaybee from product perspective would it be better to have explicit configuration similar to visibility as Codespaces do, or implicit detection would be good enough?

Reply:
Are there other ways to detect?

Reply:
ok, asking if we can reduce the scope of the PR, but if not then it is alright too 👍